tests/test_surface/test_regular_surface.py

```diff
@@ -428,68 +426,6 @@ def test_irapasc_import1_engine_compare():
     print("Python engine for read:", xtg.timer(tt0))
 
 
-def test_irapasc_export(tmpdir):
-    """Export Irap ASCII (1)."""
```

Comment:
I think maybe we should have some replacement for these tests so that the coverage does not reduce by so much. Perhaps something like: 

```python
surface_formats = ["irap_ascii", "irap_bin", "zmap_ascii"]

@pytest.mark.parametrize(
    "format1, format2",
    itertools.product(surface_formats, surface_formats)
)
def test_surface_export_and_import(tmpdir):
    x = xtgeo.RegularSurface(
        ncol=12,
        nrow=10,
        xori=1000,
        yori=5000,
        xinc=40,
        yinc=20,
        values=np.random.rand(12, 10),
    )
    
    assert x.ncol == 12
    assert x.nrow == 10
    
    file1 = join(tmpdir, "surface." + format1)
    file2 = join(tmpdir, "surface2." + format2)

    x.to_file(file1, fformat=format1)
    y = surface_from_file(fil2, fformat=format2)
    
    assert y.ncol == 12
    assert x.nrow == 10
    assert y.values.tolist() == pytest.approx(x.values.tolist())
```

Reply:
This should all be covered i: https://github.com/equinor/xtgeo/blob/ed3e99ba54f9f1cd45f75467337b51d2d32a99f3/tests/test_surface/test_file_io.py#L117

Reply:
Iooking at the coverage report, the coverage that we are missing is surface export with rotation, cube export with `rms_regular`, and well export with `rms_ascii`. All codepaths with tests we would have to change anyways. So we can do that in a separate issue.